VideoWeb/VideoWeb/Extensions/ConfigureAuthSchemeExtensions.cs

```diff
@@ -110,16 +110,16 @@ private static void AddPolicies(AuthorizationOptions options, IList<IProviderSch
             var allRoles = new[]
             {
                 AppRoles.CitizenRole, AppRoles.JudgeRole, AppRoles.RepresentativeRole, AppRoles.CaseAdminRole,
-                AppRoles.VhOfficerRole, AppRoles.JudicialOfficeHolderRole, AppRoles.StaffMember
+                AppRoles.VhOfficerRole, AppRoles.JudicialOfficeHolderRole, AppRoles.StaffMemberRole
             };
 
             var rolePolicies = new Dictionary<string, string[]>
             {
                 [AppRoles.JudgeRole] = new[] { AppRoles.JudgeRole },
                 [AppRoles.VhOfficerRole] = new[] { AppRoles.VhOfficerRole },
-                ["Judicial"] = new[] { AppRoles.JudgeRole, AppRoles.JudicialOfficeHolderRole, AppRoles.StaffMember },
-                ["Individual"] = new[] { AppRoles.CitizenRole, AppRoles.RepresentativeRole, AppRoles.QuickLinkParticipant, AppRoles.QuickLinkObserver },
```

Comment:
QuickLink users should not be removed

Reply:
I mis removed it when I fix the conflicts